src/ripple/beast/insight/impl/StatsDCollector.cpp

```diff
@@ -589,7 +589,7 @@ void StatsDGaugeImpl::flush ()
         ss <<
             m_impl->prefix() << "." <<
             m_name << ":" <<
-            m_value << "|c" <<
+            m_value << "|g" <<
```

Comment:
Just curious, what's the difference between `c` and `g` here?

Reply:
The g gets parsed as a gauge and the c gets parsed as a counter by the statsD collector.

Reply:
Ah, so this was just a bug fix :)

Reply:
Just a small bug :D